packages/atlas-figma-backup-service/.env.template

```diff
@@ -0,0 +1,5 @@
+FIGMA_EMAIL=
+FIGMA_PASSWORD=
+FIGMA_TOKEN=
+FIGMA_IDS=
+DOWNLOAD_TIMEOUT=
```

Comment:
you didn't use this env variable at all.

Reply:
right, I forgot to remove it